packages/web/src/components/Alert/Alert.scss

```diff
@@ -0,0 +1,32 @@
+@use 'theme';
+@use '../../tools/svg';
+@use '../../tools/typography';
+
+.Alert {
+    @include typography.generate(theme.$typography);
+
+    width: 100%;
+    padding: theme.$space-inner;
+    padding-left: theme.$space-inner + theme.$icon-width + theme.$space-icon;
+    margin-right: theme.$space-icon;
+    background-position: theme.$space-inner center;
+    background-repeat: no-repeat;
+    border-radius: theme.$border-radius;
+}
+
+.Alert > a,
+.Alert > strong {
```

Comment:
I'm still thinking about the `>`. Removing it would style _all_ links and strongs inside any `.Alert`, not just direct descendants. But I'm not sure if we can expect `Alert` messages to be wrapped in `div`s, list or paragraphs…